src/features/pages/options/accumulator/data.tsx

```diff
@@ -0,0 +1,279 @@
+        ),
+        content: (
+            <Link
+                target="_blank"
+                url={{
+                    type: 'non-company',
+                    href: 'https://blog.deriv.com/videos/how-to-trade-accumulator-options-on-deriv-trader/',
+                }}
+            >
+                <StaticImage
+                    src="../../../../images/common/trade-types/accumulator-video-bg-1.png"
+                    alt={localize('_t_Step-by-step guide to trade on accumulator options_t_')}
+                    loading="lazy"
+                    formats={['webp', 'auto']}
+                    quality={100}
+                    style={{ position: 'relative' }}
+                    className="cursor-pointer"
+                />
+            </Link>
+        ),
+        size: 'md',
+        align: 'start',
+        color: 'gray',
+        link: {
+            content: <Localize translate_text="_t_Learn more_t_" />,
+            href: 'https://blog.deriv.com/videos/how-to-trade-accumulator-options-on-deriv-trader/',
+            descriptiveText: 'How to trade accumulator options on deriv trader',
+        },
+    },
+    {
+        id: 2,
+        header: <Localize translate_text="_t_Read: Guide to trading accumulator options_t_" />,
+        description: (
+            <Localize translate_text="_t_Open your first accumulator options trade with our step-by-step guide._t_" />
+        ),
+        content: (
+            <StaticImage
+                src="../../../../images/common/trade-types/accumulator-video-bg-2.png"
+                alt={localize('_t_Step-by-step guide to trade on accumulator options_t_')}
+                loading="lazy"
+                formats={['webp', 'auto']}
+                quality={100}
+            />
+        ),
+        size: 'md',
+        align: 'start',
+        color: 'gray',
+        link: {
+            content: <Localize translate_text="_t_Learn more_t_" />,
+            href: 'https://blog.deriv.com/posts/a-guide-to-accumulator-options-trading-on-deriv/',
+            descriptiveText: 'A guide to accumulator options trading on deriv',
+        },
+    },
+]
+
+export const mobileCards: CardContent[] = [
+    {
+        id: 1,
+        header: <Localize translate_text="_t_1-5% growth rate_t_" />,
+        description: (
+            <Localize translate_text="_t_Choose your growth rate to tailor risk/reward. _t_" />
+        ),
+        size: 'md',
+        color: 'gray',
+        align: 'start',
+        icon: <IllustrativeMinimalCapitalIcon width="48" height="48" />,
+        className: 'h-full',
+    },
+    {
+        id: 2,
+        header: <Localize translate_text="_t_45-230 tick duration_t_" />,
+        description: (
+            <Localize translate_text="_t_Fast-paced action with durations up to 230 ticks._t_" />
+        ),
+        size: 'md',
+        color: 'gray',
+        align: 'start',
+        icon: <Image src={TickDuration} width="48" height="48" />,
```

Comment:
Don't use the atomic component... We have plan to remove atomic component and css once we completely revamp.